docs/models/renewables/wind_turbine.rst

```diff
@@ -1,37 +1,45 @@
 Wind Turbine
 ============
 
-The DISPATCHES Wind Turbine Model
+The DISPATCHES Wind Turbine Model calculates the electricity production from a wind farm using wind resource data and
+the PySAM Windpower module. The model assumes a single turbine with a powercurve from the ATB Turbine 2018 Market Average.
+Wake effects are not modeled, and PySAM's default losses are assumed. The power output of the entire farm of any size is
+calculated by scaling the output from the single turbine.
 
 Degrees of Freedom
 ------------------
 
-The Wind Turbine Model has X degrees of freedom.
+The Wind Turbine Model has 0 degrees of freedom.
```

Comment:
In this case, it is probably worth noting any parameters that need to be provided for PySAM.

Reply:
Added a sentence to Model Structure section